### PR TITLE
Make sure to checkout the correct ezplatform branch to run behat tests

### PR DESCRIPTION
**Pull request by dpobel**
https://github.com/ezsystems/PlatformUIBundle/pull/755

### CODE REVIEW

File: bin/.travis/prepare_behat.sh [hunk 1]
```diff
@@ -5,10 +5,13 @@
 # Change local git repo to be a full one as we will reuse it for composer install below
 git fetch --unshallow && git checkout -b tmp_ci_branch
 export BRANCH_BUILD_DIR=$TRAVIS_BUILD_DIR TRAVIS_BUILD_DIR="$HOME/build/ezplatform"
+
+# Checkout meta repo, use the branch indicated in composer.json under extra._ezplatform_branch_for_behat_tests
+EZPLATFORM_BRANCH=`php -r '$composer = json_decode(file_get_contents("./composer.json")); echo $composer->extra->_ezplatform_branch_for_behat_tests;'`
```
Comment (bdunogier) [inline bin/.travis/prepare_behat.sh]: Same comment than on the siblings.

**Comment by dpobel** [comment]
@andrerom TravisCI is failing on Solr tests but I hardly see the relation between the failure and my changes, is it a known issue ? can I merge this anyway ?

**Comment by andrerom** [comment]
It is known and being worked on, you can merge.

**Comment by dpobel** [comment]
@andrerom thanks :)

**Pull request merged**